File: Chronos.App/DataContracts/Members/MemberQuery.cs
using System;

namespace Chronos.App.DataContracts.Members
{
    public class MemberQuery
    {
        public string SearchText { get; set; }
        public MemberStatus[] MemberStatusId { get; set; }
        public MemberRole[] MemberRoleId { get; set; }
        public Guid[] TeamUid { get; set; }
        public Guid[] ProjectUid { get; set; }
    }
}

File: Chronos.App/ClientApp/src/app/models/member/member-query.ts
import { MemberRole, MemberStatus } from "src/app/enums";

export class MemberQuery {
    searchText = '';
    memberStatusId: MemberStatus[] = null;
    memberRoleId: MemberRole[] = null;
    teamUid: string[] = null;
    projectUid: string[] = null;

    constructor(obj?: Partial<MemberQuery>) {
        if (obj)
            Object.assign(this, obj);
    }
}
